dao/src/test/kotlin/repositories/repositoryconfiguration/DaoRepositoryConfigurationRepositoryTest.kt

```diff
@@ -109,6 +111,34 @@ class DaoRepositoryConfigurationRepositoryTest : WordSpec({
             dbEntry.shouldNotBeNull()
             dbEntry shouldBe repositoryConfig.copy(id = createdRepositoryConfiguration.id, ortRunId = ortRunId)
         }
+
+        "handle duplicates in path includes" {
+            val config = repositoryConfig.copy(
+                includes = Includes(paths = listOf(pathInclude, pathInclude))
+            )
+
+            val createdRepositoryConfiguration = repositoryConfigurationRepository.create(ortRunId, config)
+
+            val dbEntry = repositoryConfigurationRepository.get(createdRepositoryConfiguration.id)
+
+            dbEntry.shouldNotBeNull()
+            dbEntry.includes.paths shouldBe listOf(pathInclude)
```

Comment:
Use `containExcatly` and the infix version of `shouldNotBeNull`:

```kotlin
dbEntry shouldNotBeNull {
    includes.path should containExactly(pathInclude)
}
```

Same for the test below.